app/views/uploads/_satisfies.html.erb

```diff
@@ -0,0 +1,42 @@
+<span class="criteria-title">
+  <% dc_access = measure[:hqmf_document][:data_criteria] %>
+  <% dataCriteria = dc_access[reference] %>
+  <% if dataCriteria && dataCriteria[:children_criteria]%>
+
+    <% rootCriteria = dc_access[dataCriteria[:children_criteria][0]] %>
+
+    <% while rootCriteria[:children_criteria] && (!rootCriteria[:definition].include? "satisfies") do %>
+      <% rootCriteria = dc_access[rootCriteria[:children_criteria][0]] %>
+    <% end %>
```

Comment:
I'm having some trouble following this while loop. Can this please be broken out into a helper function with a comment?